### ISSUE TITLE
Click on searched Items leads not to search result

### ISSUE THREAD

**Issue by FrankMaute** [owncloud/core#23593]
### Steps to reproduce
1. Search file
2. Click on one result
3. link ends up in root folder of owncloud, not in the expected folder containing searched item

### Expected behavior
See above

### Actual behaviour
Tell us what happens instead
 link ends up in root folder of owncloud, not in the expected folder containing searched item
### Server configuration
**Operating system**:
Debian 8.3
**Web server:**
Apache 2.4
**Database:**
mySQL 5.6
**PHP version:**
5.6
**ownCloud version:** (see ownCloud admin page)
  - installed: true
  - version: 9.0.0.19
  - versionstring: 9.0.0
  - edition:
**Updated from an older ownCloud or fresh install:**
Updated
**Where did you install ownCloud from:**

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
No errors have been found.

**List of activated apps:**
Enabled:
  - activity: 2.2.1
  - calendar: 1.0
  - comments: 0.2
  - dav: 0.1.5
  - direct_menu: 0.7.1
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_pdfviewer: 0.8
  - files_pdfviewer_old: 0.7
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - imprint: 0.2.5
  - notifications: 0.2.3
  - systemtags: 0.2
  - templateeditor: 0.1
  - updatenotification: 0.1.0
  - user_external: 0.4
```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**
{
    "system": {
        "instanceid": "occ6zyjd53ts",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "xx",
            "xx.com"
        ],
        "datadirectory": "\/var\/opt\/ocdata\/data",
        "overwrite.cli.url": "https:\/\/xx.com\/owncloud",
        "dbtype": "mysql",
        "version": "9.0.0.19",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "ssl",
        "mail_from_address": "owncloud",
        "mail_domain": "xx.com",
        "mail_smtpauth": 1,
        "mail_smtpauthtype": "PLAIN",
        "mail_smtphost": "mail.your-server.de",
        "mail_smtpport": "465",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "forcessl": true,
        "app.mail.imaplog.enabled": true,
        "theme": "",
        "maintenance": false,
        "loglevel": 0,
        "log_rotate_size": 104857600,
        "appstore.experimental.enabled": true,
        "trashbin_retention_obligation": "disabled",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "localhost",
            "port": 6379,
            "timeout": 0
        },
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "updatechecker": false
    }
}
```

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No
**Are you using encryption:** yes/no
No
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No


Example: 
Search for "fahrz" leads to some results. As an example one link points to:
https://xx.com/files/index.php?dir=%2Fdocuments%2FPrivat&scrollto=FahrzeugscheinBMW.pdf

When clicking I end up in the root folder and not in the expected folder as above.

**Comment by RealRancor** [comment]
Probably a duplicate of https://github.com/owncloud/core/issues/23136 which gets fixed with oC 9.0.1 in https://github.com/owncloud/core/pull/23231

**Comment by FrankMaute** [comment]
Yes, you are right, the update solved this. Thanks for you great help :-)
